Nodejs/Product/Nodejs/NodejsProject.cs

```diff
@@ -544,7 +544,7 @@ out project
             var properties = GetExtensionObject(_innerVsHierarchy, selectionItemId).Properties;
             try {
                 var itemType = properties.Item("ItemType").Value;
-                ourEditor = (string)itemType == ProjectFileConstants.Compile && !IsES6IntellisensePreview() ? Guids.NodejsEditorFactory : Guid.Empty;
+                ourEditor = Guid.Empty;
             } catch (ArgumentException) {
                 // no item type, file is excluded from project.
                 ourEditor = Guids.NodejsEditorFactory;
```

Comment:
Also remove the itemType assignment, otherwise we might still fall back to NodejsEditorFactory. With these changes, it may also be helpful to place a TODO or track in some other way because much of this logic is actually unnecessary since we are always using the default editor